### PR TITLE
add Header and Footer commponent, set navigation and remove unused bt…

### PR DESCRIPTION
**Pull request by Wizzard26**
…n links

### CODE REVIEW

**Review by ThomasKirst — CHANGES_REQUESTED** [review]
Looks very good overall, I like the side menu.
I just found a few issues regarding Prettier formatting.

File: components/Header/index.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    .bottom {
+      transform: rotate(-45deg);
+    }
+    .middle {
+      opacity: 0;
+      width: 0;
+      margin-left: -.75rem;
+    }
+    `
+  }
```
[the remark this answers is not in the document]
Comment (Wizzard26) [reply components/Header/index.js:89]: Thanks :D

File: components/StyledLink.js [hunk 2]
```diff
@@ -42,4 +42,48 @@ export const StyledLink = styled(Link)`
...
+        &:hover {
+          color: #91AAB1;
+        }
+      `
+  }
+
+  ${({ variant }) =>
+          variant === "secondary" &&
+          css `
+        color: #E29B17;
```
[the remark this answers is not in the document]
Comment (Wizzard26) [reply components/StyledLink.js]: Prettier work a little bit different at phpstorm, i´ve edit from hand

File: components/StyledLink.js [hunk 3]
```diff
@@ -42,4 +42,48 @@ export const StyledLink = styled(Link)`
...
+  ${({ variant }) =>
+          variant === "text-light" &&
+          css `
+            color: #C6C3C3;
+
+            &:hover {
+              color: #949393;
+            }
+          `
+  }
```
[the remark this answers is not in the document]
Comment (Wizzard26) [reply components/StyledLink.js]: the same :D

File: pages/_app.js [hunk 4]
```diff
@@ -17,10 +19,17 @@ export default function App({ Component, pageProps }) {
...
+    setOpen(!open);
+  };
+
   return (
     <>
       <GlobalStyle />
+      <Header open={open} onToggle={handleToggle} />
       <Component {...pageProps} favorites={favorites} onToggleFavorite={handleToggleFavorite} />
+      <Footer />
+      {open && <div onClick={handleToggle} style={{ position: "fixed", top: 0, left: 0, bottom: 0, right: 0,zIndex:10, backgroundColor: 'rgba(32,32,32, 0.6)' }}></div>}
```
[the remark this answers is not in the document]
Comment (Wizzard26) [reply pages/_app.js]: it´s now changed to StyleComponent

**Review by ThomasKirst — APPROVED** [review]
Very good, ready to merge 👍

**Pull request merged**